templates/ts-apollo-mongodb-datasync-backend/src/index.ts

```diff
@@ -26,7 +26,7 @@ async function start() {
 
   const db = await connectDB()
 
-  const { typeDefs, resolvers, contextCreator } = createDataSyncAPI(modelDefs, { db, dataSyncConflictMap:{ Comment: { enabled: true, deltaTTL: 604800 }}});
+  const { typeDefs, resolvers, contextCreator } = createDataSyncAPI(modelDefs, { db, conflictConfig: { models: { Comment: { enabled: true } } } });
```

Comment:
Unrelated to this change but I noticed that https://github.com/aerogear/graphback/blob/c45fe8b3bc0907930fec23e418b7ed8a005d5d56/templates/ts-apollo-mongodb-datasync-backend/src/index.ts#L13 is not used anywhere. Can we clean that one up if its not too late?

Reply:
Thanks!